src/script/components/AppContainer/AppContainer.tsx

```diff
@@ -77,10 +77,7 @@ export const AppContainer: FC<AppProps> = ({config, clientType}) => {
 
   const {repository: repositories} = app;
 
-  const {isFreshMLSSelfClient, softLockLoaded = false} = useAppSoftLock(
-    repositories.calling,
-    repositories.notification,
-  );
+  const {softLockEnabled} = useAppSoftLock(repositories.calling, repositories.notification);
```

Comment:
We don't need the `loaded` info anymore since now enrollment and renewal happen at load time (which means the app is not loaded)